dart/lib/src/client.dart

```diff
@@ -77,6 +74,14 @@ abstract class SentryClient {
     return _options.transport.send(event);
   }
 
+  SentryEvent _prepareEvent(SentryEvent event) => event.copyWith(
+        serverName: event.serverName ?? _options.serverName,
+        dist: event.dist ?? _options.dist,
+        environment: event.environment ?? _options.environment,
```

Comment:
if `_options.environment` is null, it should have taken the default one which is `production`

https://github.com/getsentry/sentry-java/blob/113af8211c07ef622a4e265cc6b3438a6194d9fe/sentry/src/main/java/io/sentry/MainEventProcessor.java#L76-L79

Reply:
✅